### ISSUE TITLE
scaled svg very pixelated/blurry in Safari

### ISSUE THREAD

**Issue by Mr-Kobra** [airbnb/lottie-web#1976]
macOS Mojave
After Effects 17.0.2 (CC 2020)
newest Versions of Chrome/FF/Safari and Lottie/Bodymovin

When scaling a svg-animation in Safari (transform: scale(>1)), it will be rendered extremely pixelated/blurred. In Chrome and Firefox it renders without problems. Is this a known issue and/or is there a known workaround?

**Comment by bodymovin** [comment]
Hi, can you share a link with an example of this case?

**Comment by Mr-Kobra** [comment]
Hi,

sure, you can find it here:
http://kb.zeitweisen.com/Index.html

Like I said, in FF and Chrome it looks exactly like intended, whereas Safari shows an awful, misplaced animation.

**Comment by petertindemans** [comment]
Same problem here, it looks great in Chrome. In Safari it shows blurry.

Chrome:
<img width="579" alt="Schermafbeelding 2020-02-11 om 15 46 53" src="https://user-images.githubusercontent.com/19586163/74247060-c4ba8000-4ce5-11ea-86ac-c6d1b35d53d7.png">

Safari:
<img width="571" alt="Schermafbeelding 2020-02-11 om 15 46 48" src="https://user-images.githubusercontent.com/19586163/74247072-ce43e800-4ce5-11ea-88b3-c22329ef6d57.png">

**Comment by good-idea** [comment]
It looks like this is an issue with how Safari (mobile and desktop) deals with CSS scaling and animated SVGs. I just came across this and am now refactoring everything by setting a larger `width` property and then using translate to scoot it into place
:(

**Comment by Mr-Kobra** [comment]
@good-idea  Yes, I ended up doing the same. It's less elegant but at least it works.

**Comment by midoalone** [comment]
> I have the same problem.
> 
> Then I use object tag. It works for both image and animation.
> 
> ` <object type="image/svg+xml" data="/animated.svg">svg-animation</object>` ` <object type="image/svg+xml" data="/image.svg">svg-image</object>`
> 
> It works but not sure if it's proper. 😀

This is the only solution worked for me, thanks.

**Comment by feeshy** [comment]
> I have the same problem.
> 
> Then I use object tag. It works for both image and animation.
> 
> ` <object type="image/svg+xml" data="/animated.svg">svg-animation</object>` ` <object type="image/svg+xml" data="/image.svg">svg-image</object>`
> 
> It works but not sure if it's proper. 😀

This solution causes a new issue on some browsers --  When attached a SVG file with mask, the background color is only transparent in light mode, but solid white in dark mode.

![Light](https://github.com/user-attachments/assets/b37aba33-12f1-468a-942e-53637018d5bf) ![Dark](https://github.com/user-attachments/assets/f102e57a-d406-45eb-96a0-4a6bb6d4bfe4)

Tested on
Worked: Safari on iOS 17
Failed: Edge 126 & Firefox 128 on Windows 11, Samsung browser 25 on Android 12

**Comment by JaySoni26** [comment]
but if you use object tag the svg image loading is bad and site gets heavy & laggy
